opensrp-app/src/main/java/org/smartregister/util/Utils.java

```diff
@@ -626,7 +626,7 @@ public static String getUserInitials() {
             }
             return initials;
         }
-        catch (ArrayIndexOutOfBoundsException e)
+        catch (ArrayIndexOutOfBoundsException | StringIndexOutOfBoundsException  e)
```

Comment:
Is this exception occurring because the user does not have the required names to generate the initials?

Reply:
Caused by java.lang.StringIndexOutOfBoundsException: length=0; index=0
       at java.lang.String.charAt(String.java)
       at org.smartregister.util.Utils.getUserInitials(Utils.java:622)
       at org.smartregister.child.model.BaseChildRegisterModel.getInitials(BaseChildRegisterModel.java:199)
       at org.smartregister.child.presenter.BaseChildRegisterPresenter.updateInitials(BaseChildRegisterPresenter.java:80)
       at org.smartregister.view.activity.BaseRegisterActivity.onCreate(BaseRegisterActivity.java:95)
       at org.smartregister.child.activity.BaseChildRegisterActivity.onCreate(BaseChildRegisterActivity.java:52)
       at android.app.Activity.performCreate(Activity.java:7976)
       at android.app.Activity.performCreate(Activity.java:7965)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1306)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3505)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3689)
       at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:140)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:100)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2239)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:237)
       at android.app.ActivityThread.main(ActivityThread.java:7830)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1040)

@dubdabasoduba  as far as the exception is its occuring when this method tries to extract the user initials on empty string